src/main/java/core/basesyntax/service/impl/CsvFileWriteServiceImpl.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.Fruit;
+import core.basesyntax.service.FileWriterService;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+public class CsvFileWriteServiceImpl implements FileWriterService {
+    @Override
+    public void writeBalanceToFile(String filePath) {
+        Storage.fruits.forEach(fruit -> {
+            if (fruit == null) {
+                throw new RuntimeException("Fruit cannot be null");
+            }
+            if (fruit.getName() == null) {
+                throw new RuntimeException("Fruit name cannot be null");
+            }
+        });
```

Comment:
remove these checks

Reply:
Removed. 
Initially I added these checks in advance on that case if in the future the add() method for Fruit would be added to FruitDao interface. Then it would be possible to add null Fruit to Storage. In such case where would be better to do the check for null Fruit - in the add() method of FruitDaoImpl class or in another method like createNewFruit() of FruitServiceImpl class?

Reply:
Якщо ти береш інформацію про фрукти з файлу, то вся необхідна валідація має проводитися коли ти читаєш файл і парсиш його на список транзакцій, далі по ходу виконання программи в тебе вже ніде не може взятися `null` або інші не валідні дані